objects/029995/script.lua

```diff
@@ -74,11 +74,11 @@ function toggleObjects()
         end
         empty = false
     else
-        for _,obj in pairs(self.getObjects()) do
+        for _, bagObject in pairs(self.getObjects()) do
```

Comment:
what's up with all of these swaps?

Reply:
There is a `function(obj) .... end` a few lines below that shadows this `obj`.